internal/exec/vendor_utils.go

```diff
@@ -290,206 +264,69 @@ func ExecuteAtmosVendorInternal(
-					}
-
-					trimmedSrc := u.TrimBasePathFromPath(tempDir+"/", src)
-
-					// Exclude the files that match the 'excluded_paths' patterns
-					// It supports POSIX-style Globs for file names/paths (double-star `**` is supported)
-					// https://en.wikipedia.org/wiki/Glob_(programming)
-					// https://github.com/bmatcuk/doublestar#patterns
-					for _, excludePath := range s.ExcludedPaths {
-						excludeMatch, err := u.PathMatch(excludePath, src)
-						if err != nil {
-							return true, err
-						} else if excludeMatch {
-							// If the file matches ANY of the 'excluded_paths' patterns, exclude the file
-							u.LogTrace(cliConfig, fmt.Sprintf("Excluding the file '%s' since it matches the '%s' pattern from 'excluded_paths'\n",
-								trimmedSrc,
-								excludePath,
-							))
-							return true, nil
-						}
-					}
-
-					// Only include the files that match the 'included_paths' patterns (if any pattern is specified)
-					if len(s.IncludedPaths) > 0 {
-						anyMatches := false
-						for _, includePath := range s.IncludedPaths {
-							includeMatch, err := u.PathMatch(includePath, src)
-							if err != nil {
-								return true, err
-							} else if includeMatch {
-								// If the file matches ANY of the 'included_paths' patterns, include the file
-								u.LogTrace(cliConfig, fmt.Sprintf("Including '%s' since it matches the '%s' pattern from 'included_paths'\n",
-									trimmedSrc,
-									includePath,
-								))
-								anyMatches = true
-								break
-							}
-						}
-
-						if anyMatches {
-							return false, nil
-						} else {
-							u.LogTrace(cliConfig, fmt.Sprintf("Excluding '%s' since it does not match any pattern from 'included_paths'\n", trimmedSrc))
-							return true, nil
-						}
-					}
-
-					// If 'included_paths' is not provided, include all files that were not excluded
-					u.LogTrace(cliConfig, fmt.Sprintf("Including '%s'\n", u.TrimBasePathFromPath(tempDir+"/", src)))
-					return false, nil
-				},
-
-				// Preserve the atime and the mtime of the entries
-				// On linux we can preserve only up to 1 millisecond accuracy
-				PreserveTimes: false,
-
-				// Preserve the uid and the gid of all entries
-				PreserveOwner: false,
-
-				// OnSymlink specifies what to do on symlink
-				// Override the destination file if it already exists
-				OnSymlink: func(src string) cp.SymlinkAction {
-					return cp.Deep
-				},
-			}
+			// Log the action (handled in downloadAndInstall)
+		}
+	}
 
-			if sourceIsLocalFile {
-				if filepath.Ext(targetPath) == "" {
-					targetPath = path.Join(targetPath, filepath.Base(uri))
-				}
-			}
+	// Run TUI to process packages
+	if len(packages) > 0 {
 
-			if err = cp.Copy(tempDir, targetPath, copyOptions); err != nil {
-				return err
-			}
+		var opts []tea.ProgramOption
+		if !CheckTTYSupport() {
+			opts = []tea.ProgramOption{tea.WithoutRenderer(), tea.WithInput(nil)}
+			fmt.Println("TTY is not supported. Running in non-interactive mode.")
```

Comment:
```suggestion
			fmt.Println("No TTY detected. Falling back to basic output. This can happen when no terminal is attached or when commands are pipelined.")
```